assets/scss/components/foundation/modal/_modal.scss

```diff
@@ -61,6 +61,14 @@
     &:hover {
         color: stencilColor("icon-color-hover");
     }
+
+    &::before {
+        right: calc(50% - 0.7rem);
+    }
+
+    &::after {
+        right: calc(50% - 0.7rem);
```

Comment:
Hey @yurytut1993 , it seems that when using `remCalc(10px)` instead of `0.7rem`, the positioning of the focus tooltip isn't consistent when the element resizes:
![image](https://user-images.githubusercontent.com/19815878/136983128-1c686c77-bb62-491e-b964-32e8ebdb55eb.png)

It seems to only look right when using `right: calc(50% - 0.7rem);` instead. So I guess it _is_ a magic number, haha.
![image](https://user-images.githubusercontent.com/19815878/136983792-6ed4a564-d6a0-4161-a893-8bd29526f360.png)

Should we just stick with `calc(50% - 0.7rem)` or find a different way for it to work?